src/library/library/library.service.authorization.ts

```diff
@@ -4,7 +4,7 @@ import { ILibrary } from './library.interface';
 import { IAuthorizationPolicy } from '@domain/common/authorization-policy';
 import { LibraryService } from './library.service';
 
-@Injectable()
+Injectable();
```

Comment:
This change removes the class from the DI tree. As a result dependencies to and from the class won't be resolved, breaking the code.